### PR TITLE
(AzureCXP)  MicrosoftDocs/azure-docs-cli#2384

### PR DESCRIPTION
**Pull request by Svijay-msft**
Updated the YUM installation pivot to reflect DNF Installation

(Work in Progress)

cc: @dbradish-microsoft

### CODE REVIEW

**Comment by krish-gh** [comment]
Are we replacing yum with dnf completely? Or should both coexist?

**Comment by dbradish-microsoft** [comment]
@fengzhou-msft , Can you please see email thread with subject "Issue : 2384 | Using DNF for installing Azure CLI" and review the changes in this PR?  Thank you.

File: docs-ref-conceptual/includes/cli-install-linux-dnf.md [hunk 1]
```diff
@@ -96,8 +100,7 @@ $ sudo make install
 Finally, follow step 1 and 2 of the [install instruction](#install) to add Azure CLI repository. You can now download the package and install it without dependency.
 
 ```bash
-$ sudo yum install yum-utils -y
-$ sudo yumdownloader azure-cli
+$ sudo dnf download azure-cli
```
Comment (fengzhou-msft) [inline docs-ref-conceptual/includes/cli-install-linux-dnf.md:103]: I tried on centos 8, this command cannot be found. The command I used is `dnf install --downloadonly azure-cli --destdir .`.

**Comment by Svijay-msft** [comment]
@fengzhou-msft  -

Thank you for your response. 

Can you please confirm the release that you tried this on ? I'll try checking the same at my end.

When I had tried in my CentOS lab system with the below config - OS Source downloaded from this[ source](http://isoredirect.centos.org/centos/8/isos/x86_64/)


image

Release info : 
image

I was able to download using the `sudo dnf download azure-cli`

image


`dnf install --downloadonly azure-cli`

image

Let me know if I am missing something / doing incorrectly ?

**Comment by fengzhou-msft** [comment]
I tested in a docker image with  `docker run -it --rm centos:8`.

**Comment by Svijay-msft** [comment]
@fengzhou-msft - I'll try at my end as well. Meanwhile, just a quick question if I may for my own understanding - since it is working in the out of box Cent OS 8 - can we safely assume the commands used in the PR are correct or further testing is required ?

**Comment by Svijay-msft** [comment]
@fengzhou-msft ,  
When I tested using docker container. I was able to install using the above mentioned command without any issues :

`dnf install --downloadonly azure-cli`

I got command not found for `dnf download`   when I ran `dnf download azure-cli`

I had ran the below command to resolve the issue :

`dnf install 'dnf-command(download)'`

I was able to run the 

`dnf download azure-cli`

Can I add a note to include the command  `dnf install 'dnf-command(download)'`  for command not found ?

**Comment by fengzhou-msft** [comment]
Yes, please add the install command.

File: docs-ref-conceptual/includes/cli-install-linux-dnf.md [hunk 2]
```diff
@@ -35,21 +35,25 @@ for the Azure CLI. This package has been tested with RHEL 7.7, RHEL 8, Fedora 24
...
-   sudo yum install azure-cli
+   sudo dnf install azure-cli
    ```
-
+ 
 Run the Azure CLI with the `az` command. To sign in, use [az login](/cli/azure/reference-index#az-login) command.
+ 
+> [!NOTE]
+>
+> Use `yum` package manager if you are using Linux systems that don't support `dnf` package manager.
```
Comment (fengzhou-msft) [inline docs-ref-conceptual/includes/cli-install-linux-dnf.md:48]: Can we move up this note under `Overview`, probably below the `Important` notice?

Comment (Svijay-msft) [reply docs-ref-conceptual/includes/cli-install-linux-dnf.md:48]: Sure. I will move it under the overview section.

**Comment by Svijay-msft** [comment]
@BhargaviAnnadevara-MSFT

**Review by fengzhou-msft — APPROVED** [review]

File: docs-ref-conceptual/includes/cli-install-linux-dnf.md [hunk 3]
```diff
@@ -35,21 +39,22 @@ for the Azure CLI. This package has been tested with RHEL 7.7, RHEL 8, Fedora 24
    gpgkey=https://packages.microsoft.com/keys/microsoft.asc" | sudo tee /etc/yum.repos.d/azure-cli.repo
```
Comment (BhargaviAnnadevara-MSFT) [inline docs-ref-conceptual/includes/cli-install-linux-dnf.md:39]: @Svijay-msft Should this still be `yum.repos.d`?

Comment (Svijay-msft) [reply docs-ref-conceptual/includes/cli-install-linux-dnf.md:39]: DNF -  by default uses all *.repo files found under /etc/yum.repos.d. So we should be good having **yum.repos.d**  .

**Review by BhargaviAnnadevara-MSFT — APPROVED** [review]

**Comment by dbradish-microsoft** [comment]
With @fengzhou-msft 's approval acquired, I am merging this PR.

**Pull request merged**